### PR TITLE
Remove a fixme in ClearAOCSFileSegInfo()

### PR DESCRIPTION
**Pull request by adam8157**
It looks similar now, I checked the codes before and after the merge.
Remove the FIXME.

### CODE REVIEW

**Comment by JunfengYang** [comment]
There is a diffenence that `ClearFileSegInfo` using `table_beginscan_catalog` to scan the aoseg relation, but `ClearAOCSFileSegInfo` is using `systable_beginscan`. Also for the `heap_endscan` and `systable_endscan`.

**Comment by adam8157** [comment]
> There is a diffenence that `ClearFileSegInfo` using `table_beginscan_catalog` to scan the aoseg relation, but `ClearAOCSFileSegInfo` is using `systable_beginscan`. Also for the `heap_endscan` and `systable_endscan`.

Thanks for review, I made a fixup.

File: src/backend/access/aocs/aocssegfiles.c [hunk 1]
```diff
@@ -681,7 +672,7 @@ ClearAOCSFileSegInfo(Relation prel, int segno)
 	pfree(newtup);
 	pfree(vpinfo);
 
-	systable_endscan(scan);
+	heap_endscan(scan);
```
Comment (JunfengYang) [inline src/backend/access/aocs/aocssegfiles.c:675]: Is it better to use `table_endscan` which call the AM just like `table_beginscan_catalog`?

**Review by JunfengYang — APPROVED** [review]
+1

File: src/backend/access/aocs/aocssegfiles.c (hunk 1 above)
Comment (adam8157) [reply src/backend/access/aocs/aocssegfiles.c]: It's the same thing here, but it looks better, I made another fixup.

**Pull request merged**